overlord/snapstate/backend_test.go

```diff
@@ -45,6 +45,37 @@ type fakeOp struct {
 	old string
 }
 
+type fakeOps []fakeOp
+
+func (ops fakeOps) Ops() []string {
+	opsOps := make([]string, len(ops))
+	for i, op := range ops {
+		opsOps[i] = op.op
+	}
+
+	return opsOps
+}
+
+func (ops fakeOps) Count(op string) int {
```

Comment:
yep! used in the aba branch for one (this is part of my whittling the aba branch)